common/config/rush/common-versions.json

```
@@ -69,7 +69,7 @@
     // @azure/event-processor-host is on a much lower major version
     "@azure/ms-rest-nodeauth": ["^0.9.2"],
     // Idenity is moving from v1 to v2. Moving all packages to v2 is going to take a bit of time, in the mean time we could use v2 on the perf-identity tests.
-    "@azure/identity": ["^2.0.0-beta.1", "2.0.0-beta.3", "~1.3.0"],
+    "@azure/identity": ["2.0.0-beta.2", "2.0.0-beta.3", "~1.3.0"],
```

So that we'll have an allowed version that's glued to 2.x master if we decide to do a beta.4 for any reason.

```suggestion
    "@azure/identity": ["2.0.0-beta.2", "^2.0.0-beta.3", "~1.3.0"],
```

---

That would work only if in the dependencies we use `^2.0.0-beta.3` right? But we don't want to do that because we try to pin preview dependencies